src/connector/src/parser/unified/avro.rs

```diff
@@ -269,6 +270,13 @@ impl<'a> AvroParseOptions<'a> {
             (Some(DataType::Jsonb), Value::String(s)) => {
```

Comment:
Is this branch actually unreachable? It is inconsistent with the new branch below:
* In this branch, `abc` is invalid and `{"foo": 1}` becomes `'{"foo": 1}'::jsonb` in RisingWave.
* In the branch below, `abc` (as value of `map<string>`) becomes `'"abc"'::jsonb` and `{"foo": 1}` becomes `'"{\"foo\": 1}"'::jsonb`.

Reply:
I was also thinking whether this is unreachable. From `avro_type_mapping`, there seems no `JSONB` used before..

Reply:
> `abc` (as value of `map<string>`)

I don't get this. How is `abc` a `map<string>`?

Reply:
I removed this path

Reply:
> > `abc` (as value of `map<string>`)
> 
> I don't get this. How can `abc` be a `map<string>`?

By `value` I meant key-value of a map, not a value of a type (here `map`).